src/modules/flickr-api.js

```diff
@@ -0,0 +1,41 @@
+
+const  setCityPhoto = (cityName) => {
+
+const url = 'https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key='+
+                process.env.FLICKR_API_KEY+'&tags=Skyline&text='+cityName.replace(' ','+')
+                +'&sort=relevance&accuracy=11&content_type=1machine_tag_mode=OR&format=json&nojsoncallback=1';
+
+// Tags City,Urban,Downtown,Town,Block,Street, Panoramic
+      getData(url);
+}
+const getCityPhotoByLatLong = (lat,long) => {
+
+}
+
+
+const setRandomCityPhoto = () => {
+  const cities = ['New York City','San Francisco','Las Vegas','Dubai','Berlin','Tokyo']
+  const randomIndex =  Math.round(Math.random() * (cities.length - 0) + 0);
+  console.log(cities[randomIndex]);
+  getCityPhotoUrl(cities[randomIndex]);
+
+}
+const getData = (url) => {
+  fetch(url)
```

Comment:
This should be using `async/await or promises`, we don't know when the data will return